### DISCUSSION TITLE
mspider/user-guide/gateway-instance/create/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1308] [Announcements]
# mspider/user-guide/gateway-instance/create/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/gateway-instance/create/

**Comment by liyuerich** [comment]
还有新选项 工作负载标签

**Comment by windsonsea** [comment]
Fixed in https://github.com/DaoCloud/DaoCloud-docs/pull/1311

**Comment by nicole-lihui** [comment]
添加了弹性伸缩，图片需要更新
